punx/tests/test_issue130.py

```diff
@@ -0,0 +1,58 @@
+"""
+Issue 130: BUG: tree cannot parse some NeXus examples
+"""
+
+import h5py
+import os
+
+from ._core import EXAMPLE_DATA_DIR
+from ._core import hfile
+from .. import h5tree
+
+DLS_EXAMPLE_FILE = "DLS_i03_i04_NXmx_Therm_6_2.nxs"
+
+TESTFILE = os.path.join(EXAMPLE_DATA_DIR, DLS_EXAMPLE_FILE)
+
+
+def test_DLS_master_file():
+    os.path.exists(TESTFILE)
+
+    tree = h5tree.Hdf5TreeView(TESTFILE)
+    assert tree.isNeXus
+
+    report = tree.report()
+    assert isinstance(report, list)
+    assert len(report) == 183
+
+
+def test_SwissFEL_file_replica(hfile):
```

Comment:
```suggestion
def test_SwissFEL_file_replica(hfile):
"""Test that missing external links are handled gracefully.

Create a NeXus file with a broken external link and an internal link to that broken link. Check
that the missing file is correctly reported but does not raise an exception.
"""
```
Something like this would be more helpful because this test isn't about file replication.